src/src/rx_main.cpp

```diff
@@ -1723,6 +1744,23 @@ void loop()
     updateSwitchMode();
     checkGeminiMode();
     debugRcvrLinkstats();
+
+#if defined(DEBUG_RCVR_DUAL_RSSI)
+    if(now - lastReport >= 1000)
+    {
+        //DBGLN("IRQ counts: %d, %d / telem: %d, %d / dio but failed: %d"
+        DBGLN("%d\t%d\t%d\t%d\t%f\t%f\t%d\t%d\t%d"
```

Comment:
What about collecting the min/max SNR for each radio as well. That might help us further with our stats collection and be useful for type approval of dual radio systems. We could even extend it to capture that for single radio devices too as it could be useful for them too.

Reply:
Haven't tested on a single radio but in principle, it'll work as it is I believe

Reply:
* Single radio mode is added
* Max SNR is added (no min snr because it's almost always the minimum possible value, which is not very useful)